### PR TITLE
Document V2 in master README

### PR DESCRIPTION
**Pull request by rtfb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,31 +12,52 @@ punctuation substitutions, etc.), and it is safe for all utf-8
...
 ------------
 
-Blackfriday is compatible with Go 1. If you are using an older
-release of Go, consider using v1.1 of blackfriday, which was based
-on the last stable release of Go prior to Go 1. You can find it as a
-tagged commit on github.
-
-With Go 1 and git installed:
+Blackfriday is compatible with any modern Go release. With Go 1.7 and git
+installed:
```
Comment (shurcooL) [inline README.md]: This is a little confusing. First it says "any modern Go release", then "Go 1.7". Can we be more clear?

File: README.md [hunk 2]
```diff
@@ -8,31 +12,52 @@ punctuation substitutions, etc.), and it is safe for all utf-8
...
 
-    go get github.com/russross/blackfriday
+    go get gopkg.in/russross/blackfriday.v2
 
 will download, compile, and install the package into your `$GOPATH`
 directory hierarchy. Alternatively, you can achieve the same if you
```
Comment (shurcooL) [inline README.md]: This "Alternatively" section should probably be removed. It's easier to tell people to `go get -u` the import path, they can figure out the rest by now.

File: README.md [hunk 3]
```diff
@@ -8,31 +12,52 @@ punctuation substitutions, etc.), and it is safe for all utf-8
...
 import it into a project:
 
-    import "github.com/russross/blackfriday"
+    import "gopkg.in/russross/blackfriday.v2"
 
 and `go get` without parameters.
```
Comment (shurcooL) [inline README.md]: Is it intentional that the v1 import path is not mentioned at all in the README?

Comment (rtfb) [inline README.md]: No, fixed.

**Comment by Ambrevar** [comment]
The README should be updated. More updates are needed to reflect the removal of MarkdownBasic and MarkdownCommon, the addition of the Run function, the Markdown type, etc.

**Comment by rtfb** [comment]
TODO: merge #377 here.

**Comment by rtfb** [comment]
#292 can be closed when this lands.

**Pull request merged**